<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>indiceDialog</class>
 <widget class="QDialog" name="indiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="aLabel">
       <property name="text">
        <string>A:  </string>
       </property>
       <property name="buddy">
        <cstring>aSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="aSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="bLabel">
       <property name="text">
        <string>B:  </string>
       </property>
       <property name="buddy">
        <cstring>bSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="bSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cLabel">
       <property name="text">
        <string>C:  </string>
       </property>
       <property name="buddy">
        <cstring>cSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="cSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aSpinBox</tabstop>
  <tabstop>bSpinBox</tabstop>
  <tabstop>cSpinBox</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>Delete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
